### PR TITLE
[RAPTOR-4105] add reading params from env vars

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
- make drum able to read params from env vars
- remove now redundant transition from env vars to cli args in start_server.sh
- set `/opt/code/start_server.sh` as an entry point in the public env

## Rationale
This is done to make CM PPS docker usage easier when starting container from either CLI or docker compose, e.g

`docker  run -e TARGET_TYPE=regression cm_pps_image`

### CODE REVIEW

File: custom_model_runner/datarobot_drum/drum/common.py [hunk 1]
```diff
@@ -175,6 +175,32 @@ class ArgumentsOptions:
...
+    VARS_WITH_VALUES = [
+        TARGET_TYPE,
+        CODE_DIR,
+        NEGATIVE_CLASS_LABEL,
+        POSITIVE_CLASS_LABEL,
+        CLASS_LABELS_FILE,
+        ADDRESS,
+        MAX_WORKERS,
+    ]
+    BOOL_VALUES = [WITH_ERROR_SERVER, SHOW_STACKTRACE, PRODUCTION, MONITOR]
```
Comment (rvorobii) [inline custom_model_runner/datarobot_drum/drum/common.py:201]: nit
VARS_WITH_VALUES -> VALUE_VARS
BOOL_VALUES           -> BOOL_VARS

**Review by rvorobii — APPROVED** [review]
LGTM:+1:

File: custom_model_runner/datarobot_drum/drum/common.py (hunk 1 above)
Comment (yakov-g) [reply custom_model_runner/datarobot_drum/drum/common.py:201]: ok!

Comment (rvorobii) [reply custom_model_runner/datarobot_drum/drum/common.py:201]: (it's just the first one is called VARS_... and the second one is called ..._VALUES)

Comment (rvorobii) [reply custom_model_runner/datarobot_drum/drum/common.py:201]: the suggested renaming makes the naming a bit more consistent imo

Comment (yakov-g) [reply custom_model_runner/datarobot_drum/drum/common.py:201]: no problem!

**Review by meltzg — APPROVED** [review]

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**27.11** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**